src/analysis/ContextBar.ts

```diff
@@ -82,9 +82,9 @@ export interface ContextBarProps {
   runtimes: Runtime[];
   apps: App[];
   appDataDisks: PersistentDisk[];
-  refreshRuntimes: (maybeStale?: boolean) => Promise<void>;
+  refreshRuntimes: (maybeStale?: boolean) => Promise<unknown>;
```

Comment:
I think `Promise<void>` makes more sense for these. They're intended to kick off a refresh but not return anything.